roles-tag-and-release.sh

```diff
@@ -106,7 +106,21 @@ if [ "$skip" = false ]; then
         ${EDITOR:-vi} "$rel_notes_file"
         if [ -f CHANGELOG.md ]; then
             read -r -p "Edit CHANGELOG.md - press Enter to continue"
-            cat "$rel_notes_file" CHANGELOG.md > .tmp-changelog
+            pat="=*"
+            if [[ $( head -n 1 CHANGELOG.md ) == "# Changelog" ]]; then
```

Comment:
Why do we have two different styles of changelog file?  We should have only 1

Reply:
If we don't have to support
```
Changelog
=========
```
I'm more than happy! I'm eliminating it and just support `# Changelog`.
Thanks!

Reply:
note that the network role is the only role that currently has a CHANGELOG and it uses
```
Changelog
=========
```
What if we only use that form?  Is there somewhere else that uses the `# Changelog` form?

Reply:
Well, I already generated the initial CHANGELOG.md for most of the roles using `github-changelog-generator`, which generates `# Changelog` form...

Reply:
ok - I think the network role shouldn't mind changing the `====` form to the `# Changelog` form - either way is fine

Reply:
Fingers crossed... May I ask them if they don't mind?

Reply:
> Fingers crossed... May I ask them if they don't mind?

Sure